framework/src/main/java/org/tron/core/zen/ZksnarkInitService.java

```diff
@@ -24,11 +24,6 @@ private void init() {
   public static void librustzcashInitZksnarkParams() {
```

Comment:
Can this be optimized so that it is only initialized once in the JVM? Currently, a large number of unit tests repeatedly generate related files, causing insufficient space in the tmp folder.

<img width="680" height="153" alt="Image" src="https://github.com/user-attachments/assets/6d608456-dfb6-4792-91b4-3ea6bd07fc21" />

<img width="681" height="382" alt="Image" src="https://github.com/user-attachments/assets/a3cde265-6260-4b50-b57b-86489a30ff76" />

Reply:
ok, I will fix it.